c++/src/kj/compat/http.c++

```diff
@@ -1637,6 +1667,10 @@ public:
     return { headerBuffer.releaseAsBytes(), leftover.asBytes() };
   }
 
+  kj::Promise<uint64_t> pumpTo(AsyncOutputStream &output, size_t maxBytes) {
```

Comment:
I'm concerned this method could easily be mistaken as being consistent with `tryRead()` above -- i.e. in any case where you want to use `tryRead()`, you might expect you can also use `pumpTo()` and get the same data out. But this isn't the case, they are not consistent. Technically since this class doesn't implement `AsyncInputStream`, there's no actual need for these methods to be consistent, but someone could easily accidentally assume that the interface here is meant to correspond to `AsyncInputStream`.

Since this method is _only_ used to discard bytes, could we change it to just do that, and call it `discard(uint64_t)`?